aiosonos/api/websockets.py

```diff
@@ -188,7 +188,7 @@ def _handle_incoming_message(
         # handle error message
         if "errorCode" in msg_data:
             if "cmdId" not in msg:
-                self.logger.error("Received error message without cmdId: %s", msg)
+                self.logger.error("Received error message without cmdId: %s, msg_data=%s", msg, msg_data)
```

Comment:
```suggestion
                self.logger.error("Received unhandled error: %: %s", msg, msg_data)
```

Reply:
Lets shorten it a bit - this error will get thrown to catch any errors that are thrown without us sending a specific command.

Reply:
Of course! I couldn't test it properly, and now I see a missing %s in % :-( Hold on.